instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/BaseTracer.java

```diff
@@ -36,13 +38,32 @@
       ContextKey.named("opentelemetry-trace-auto-client-span-key");
 
   protected final Tracer tracer;
+  protected final ContextPropagators propagators;
 
   public BaseTracer() {
     tracer = GlobalOpenTelemetry.getTracer(getInstrumentationName(), getVersion());
+    propagators = GlobalOpenTelemetry.getPropagators();
   }
 
+  /**
+   * Prefer to pass in an OpenTelemetry instance, rather than just a Tracer, so you don't have to
+   * use the GlobalOpenTelemetry Propagator instance.
+   *
+   * @deprecated prefer to pass in an OpenTelemetry instance, instead.
```

Comment:
These artifacts aren't used much so not a big deal IMO, but as general practice, we'd link to the new method, `@deprecated Use {@link #BaseTracer(OpenTelemetry)} instead`

Reply:
that's a good idea. Happy to amend this PR, or follow-up later.